.release-please-manifest.json

```diff
@@ -1,3 +1,3 @@
 {
-  ".": "0.9.2"
```

Comment:
Never modify this file its auto managed, when we do the next release it will be flagged as a major change and do a major version bump.